readyplayerme/rpm-unity-sdk-core
# feat: update avatar creator simple scene to contain user management, create, and save avatar functionality

## [SDK-683](https://ready-player-me.atlassian.net/browse/SDK-683)

![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/107070960/6d94a274-6672-41c0-a763-0e831cc68946)


## Description

-   Update the avatar creator scene, so that you can log in/log out, and create and save new avatars from the template.

## How to Test

-   Verify, that everything works





[SDK-683]: https://ready-player-me.atlassian.net/browse/SDK-683?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
